update-dependencies/NuGet.config

```diff
@@ -2,5 +2,6 @@
 <configuration>
   <packageSources>
     <add key="dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
+    <add key="dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/" />
```

Comment:
iirc, this url ends up using the v2 NuGet feed. v3 should work better.

Reply:
Fixed